ui/candidate-portal/src/app/components/account/login/login.component.ts

```diff
@@ -71,13 +71,14 @@ export class LoginComponent implements OnInit {
     if (this.loading) { return; }
     this.loading = true;
 
-    const action = 'login';
-    this.reCaptchaV3Service.execute(action).subscribe(
-      (token) => this.loginWithToken(token),
-      (error) => {
-        console.log(error);
-      }
-    );
+    // const action = 'login';
+    // this.reCaptchaV3Service.execute(action).subscribe(
+    //   (token) => this.loginWithToken(token),
+    //   (error) => {
+    //     console.log(error);
+    //   }
+    // );
```

Comment:
Same comment here as with the admin-portal login - bypassing recaptcha - is that intentional?